ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveUtils.java

```diff
@@ -543,4 +546,10 @@ public static String getLowerCaseTableName(String refName) {
     }
     return refName.toLowerCase();
   }
+
+  public static String getCurrentCatalogOrDefault(Configuration conf) {
+    return SessionState.get() != null ?
```

Comment:
you could try Optional.ofNullable

Reply:
Fixed in https://github.com/apache/hive/pull/6088/commits/00a75f9fe1d13cc9e49a00a3cd2b598dadef1f0b